test/src/sdk-harness/test_projects/exponentiation/mod.rs

```diff
@@ -1,19 +1,17 @@
 use fuels::prelude::*;
+use fuels::signers::LocalWallet;
 use fuels::tx::ContractId;
 use fuels_abigen_macro::abigen;
-use fuels::signers::LocalWallet;
 
-abigen!(
-    TestPowContract,
-    "test_artifacts/pow/out/debug/pow-abi.json"
-);
+abigen!(TestPowContract, "test_artifacts/pow/out/debug/pow-abi.json");
```

Comment:
Shouldn't the exponentiation code be another issue?

Reply:
All I did was run `cargo fmt`. I will revert this file for now.